perf/benchmark.jl

```diff
@@ -1,65 +1,23 @@
-ca_dir = joinpath(dirname(@__DIR__));
```

Comment:
Changes in this file look accidental? We'll definitely want to revert this

Reply:
Absolutely. I had just noticed that something was mingled in the rebase and I was just fixing this now. Sorry about that